### ISSUE TITLE
Add workspace failed in smartIDE server , which is in deploy state.

### ISSUE THREAD

**Issue by neblen** [SmartIDE/SmartIDE#38]
Add workspace failed in smartIDE server , which is in deploy state.
I deploy smartIDE server by using standalone installation script.Installed successfully but cannot run the workspace
![image](https://user-images.githubusercontent.com/49058548/197731027-f7cf0654-5bbc-4b5d-a2fc-8675685043e1.png)

**Comment by ups216** [comment]
did you use root user doing the deployment?

**Comment by neblen** [comment]
> did you use root user doing the deployment?

Yes! I deployed with the super administrator user, but the single node deployed is a virtual machine on vmware.
